ci/travis/cover.sh

```diff
@@ -2,6 +2,12 @@
 
 set -ex
 
+### PyPy does not run the test suite ###########################################
+
+if ! [[ $FEATURES == *"pypy"* ]]; then
```

Comment:
Did I miss something? Why isn't the test suite run in Pypy?

Reply:
It is currently impossible to initialize pypy from rust, and call cpyext (PyPy doesn't  expose the `PyInitialize` symbol).
If you'll try to run a test binary it will immediately segfault.

However if we could expand the test suite that uses rust from PyPy we could hit more code paths, and that is a good start :)

Reply:
Hm.. I guess it doesn't have to block things, but I am personally uncomfortable with the idea of skipping the *entire* test suite on PyPy, we've already seen one regression, and the `rustapi_module` tests only cover a small fraction of the library.

Reply:
Simple tests in Python can be run using pytest. I would appreciate that being done, because when I tried this branch on a project, the build succeeded but Python tests segfaulted.

Reply:
@ijl do you know what failed? can we add it to the test suite?